dbms/src/Storages/Page/V1/PageFile.cpp

```diff
@@ -58,7 +59,7 @@ using Checksum = UInt64;
 static const size_t PAGE_META_SIZE = sizeof(PageId) + sizeof(PageTag) + sizeof(PageOffset) + sizeof(PageSize) + sizeof(Checksum);
 
 /// Return <data to write into meta file, data to write into data file>.
-std::pair<ByteBuffer, ByteBuffer> genWriteData( //
+std::pair<std::span<char>, std::span<char>> genWriteData( //
```

Comment:
I wonder what is the benefit of `std::span` over `ByteBuffer`?

Reply:
Oh, details are documented here: https://github.com/pingcap/tiflash/pull/6972